### ISSUE TITLE
Add a friendly method name for gevent

### ISSUE THREAD

**Issue by tianquan23** [svinota/pyroute2#360]
The gevent environment conflicts with socket in some way like eventlet. Thanks to eventlet_config(), pyroute2.NetNS could also be used with gevent. However, it would lead to misunderstanding if we call eventlet_config in the gevent context. I was wondering if we could add a friendly method name for gevent. pyroute2.config.gevent.gevent_config might be good enough. Thanks!

**Comment by svinota** [comment]
Yep, sure. Do I understand correctly that it is enough to have a `gevent_config` as an alias to the same code, as in `eventlet_config`?

Regarding the eventlet library, they completely broke the API, so I'm not sure how it will work with eventlet from now on.

But if I can provide some support for gevent, that's great. If the gevent team will not break the API thereafter :)

**Comment by tianquan23** [comment]
Thank you so much! And yes, the existing code is good for gevent as far as I know, and an alias is indeed what I was expecting. However, frankly speaking, I have only verified that NetNS would be fixed by applying eventlet_config, which I think is the only place being conflict with gevent. Is that right? 

If there might be more potential conflicts, I‘m glad to do more verification, or provide some unit tests if necessary.

**Comment by svinota** [comment]
Moved the function to `pyroute2.config.asyncio`:
```python
from gevent import monkey
from pyroute2.config.asyncio import asyncio_config
from pyroute2 import NetNS

monkey.patch_all()
asyncio_config()

with NetNS('test') as ns:
    print(ns.get_links())
```

The code works, but I have got problems running functional tests via `nose`. I'll need your help, probably — I don't work with gevent and not sure how to debug it. I'll provide the branch with the tests a bit later.

**Comment by tianquan23** [comment]
I see you have merged the code. It' great. Thanks!
Is there still problems running functional tests? It's my pleasure if I can do anything for this.
